.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -3,15 +3,27 @@ name: 🎉 Feature request
 about: Suggest an idea to improve GitHub Desktop
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I have an issue when [...]
+## Goal
+A clear and concise description of what the problem and request are. Ex. I have an issue when [...problem...]. I'd like [...feature...] so that [...reason...]
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen. Add any considered drawbacks.
+#### Background
+Any related issues or any preceding features that this would depend on. (linked from desktop repository)
 
-**Describe alternatives you've considered**
+## Solution
+A clear and concise description of what you want to happen, implementation-wise. Add any considered drawbacks. If you can, explain how users will be able to use this.
+
+#### Alternatives Considered
 A clear and concise description of any alternative solutions or features you've considered.
 
-**Teachability, Documentation, Adoption, Migration Strategy**
-If you can, explain how users will be able to use this. This will help the core team understand how the feture would work, and assist with future work like documentation.
-Screenshots or mockups here are a great help!
\ No newline at end of file
+## Teachability, Documentation, Adoption, Migration Strategy*
+This will help the core team understand how the feature would work, and assist with future work like documentation.
+Screenshots or mockups here are a great help! Here are some example subheaders:
+
+#### Task List
+- [ ] a
+- [ ] b
+- [ ] c
+- [ ] incorporate analytics
+
+#### Mockups

```

Comment:
Personal feels: I'd move this up below the `Solution` section as this feels like they both belong together.

Reply:
i'm open to considering; the only reason i had them at the end was that i feel like no one reads after the mockups since they can get so long and i wanted the text together. 
do you read much beyond? if so, maybe it's just a worry that isn't grounded in anything real!

Reply:
> i'm open to considering; the only reason i had them at the end was that i feel like no one reads after the mockups since they can get so long and i wanted the text together.

Yeah, I can see the side of that. Alternatively, if mockups become significant they could be done in their own comment after, with details there. I guess I'd ask how much detail we need to  capture in the feature request template at the start? We might have a couple of mockups initially for something - those might help alongside the words. Someone might then have more detailed mocks - I guess those could be added to a follow-up comment with their own words to separate them better.

> do you read much beyond? 

Initially, probably not. But having the words alongside the text might help better for wrapping my head around a feature in terms of the flow - moving mockups away from the words about the feature that might be confusing?